CNAME

```diff
@@ -1 +0,0 @@
-limetext.org
```

Comment:
Last I checked, the CNAME is needed for proper GitHub Pages DNS routing. Has that changed?